static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -188,6 +188,12 @@ define([
             }
         });
 
+        if (window.location.hash.includes('devtools')) {
```

Comment:
The quote I put was wrong btw, we don't polyfill like that, we do it with https://www.npmjs.com/package/babel-plugin-transform-runtime which states 

// instance methods such as "foobar".includes("foo") will not work since that would require modification of existing builtins

So it wouldn't work anyway? :man_shrugging: